panel/param.py

```diff
@@ -772,15 +772,22 @@ class ParamMethod(ReplacementPane):
         Whether to lazily evaluate the contents of the object
         only when it is required for rendering.""")
 
-    loading_indicator = param.Boolean(default=False, doc="""
-        Whether to show loading indicator while pane is updating.""")
+    loading_indicator = param.Boolean(default=None, doc="""
+        Whether to show a loading indicator while the pane is updating.
+        Can be set as parameter or by setting panel.config.loading_indicator.""")
 
     def __init__(self, object=None, **params):
         if (
-            self.param.defer_load.default is None and
-            'defer_load' not in params and config.defer_load
+            'defer_load' not in params and
```

Comment:
I aligned the `defer_load` and `loading_indicator` code. I prefer the parameters to be set to `True` or `False`. But I don't know if it was on purpose that `None` is kept in the original implementation.

Reply:
I actually think it might be easier if we simply have the `config` set the default value on the ParamMethod class, i.e. on `pn.config` we'd have a callback like:

```python
@param.depends('defer_load', watch=True)
def _update_defer_load_default(self):
    ParamMethod.defer_load = self.defer_load
```

@param.depends('loading_indicator', watch=True)
def _update_loading_indicator_default(self):
    ParamMethod.loading_indicator = self.loading_indicator
```